container/cli.py

```diff
@@ -191,6 +195,9 @@ def commandline():
         LOGGING['loggers']['container']['level'] = 'DEBUG'
     config.dictConfig(LOGGING)
 
+    if args.playbook:
+        args.playbook = os.path.realpath(args.playbook)
```

Comment:
Not sure you want to do this, as it needs to be evaluated in the context of the build container.